### PR TITLE
made a check box labeled it done for now

### PR DESCRIPTION
**Pull request by chloedeveloper**
This is the CheckBox functionality. You can click and unclick each task.

Future plans for the trello board could be adding a strike through the done tasks.

Beware there is a warning, and I have not pulled the latest pull request into mine as well. Please help me figure out how to get rid of said warning.

Branch Name: addCheckBox

### CODE REVIEW

**Comment by chloedeveloper** [comment]
Test plan:

Check out branch addCheckBox
Pull latest code
Do npm install
Do npm start
Verify app loads locally
Add a task to the list
Add another task from the list
Click the first check box to show as done
Unclick the first checkbox to undo

File: src/ListComponent.js [hunk 1]
```diff
@@ -8,9 +8,12 @@ import InputGroup from "react-bootstrap/InputGroup";
 import FormControl from "react-bootstrap/FormControl"; 
 import ListGroup from "react-bootstrap/ListGroup"; 
 
+
 const ListComponent = (props) => { 
     const [input, setInput] = useState('');
     const [list, setList] = useState([]);
+    const [done, complete] = useState(true);
```
Comment (ectherrien) [inline src/ListComponent.js:15]: I am new to react/js so take this with a grain of salt...

I think this should be inside the taskToAdd object, this way if a task is complete it is stored with all of that task's values/variables. See next comment for what I mean sorta.

Comment (ectherrien) [reply src/ListComponent.js:15]: Well apparently GitHub does not allow you to add a comment to an unchanged line...annoying or am I using it wrong?

I meant to add like `done: false,` to new line #22

File: src/ListComponent.js [hunk 2]
```diff
@@ -94,6 +112,7 @@ const ListComponent = (props) => {
                                                     onClick={() => editTask(id)}> 
                                                     Edit 
                                                 </Button> 
+                                              <input type = "checkbox" value={done} onChange={()=>handleChange("done")} /> done
```
Comment (ectherrien) [inline src/ListComponent.js:115]: So here I would think onChange would just cycle the item.done value...unsure on exact syntax but something like `item.done = ! item.done. And I think value would be item.done in this case too?

File: src/ListComponent.js [hunk 3]
```diff
@@ -32,6 +35,21 @@ const ListComponent = (props) => {
         /* add here */
     } 
 
+    const handleChange = (id)=>{
```
Comment (ectherrien) [inline src/ListComponent.js:38]: Is this needed with the suggested changes? Honestly not sure, might be able to get rid of it?

**Comment by ectherrien** [comment]
Overall this does work, my comments are more like implementation comments. I could be wrong, but I think the changes I am suggesting might make it easier in the future to update the GUI...however I do not have strong opinions, I will let it be up to @pragatipant since I think she's more of a React/JS expert :)

**Comment by pragatipant** [comment]
Suggested changes will be addressed as part of [Filter Tasks user story](https://trello.com/c/N4bmRvGV).

**Review by pragatipant — APPROVED** [review]

**Pull request merged**